### ISSUE TITLE
elementsOf(Iterable) for contains.inAnyOrder.only

### ISSUE THREAD

**Issue by robstoll** [robstoll/atrium#127]
*Platform* (jvm, jdk8, js, android): all

## Code related feature
```
val otherIterable : Iterable<Int>
expect(listOf(1, 2)).contains.inAnyOrder.only.elementsOf(otherIterable)

//instead of

expect(listOf(1, 2)).contains.inAnyOrder.only.elementsOf(otherIterable.first(), *otherIterable.drop(1).toTypedArray())
```

Following the things you need to do (see also a0b539c0 for a guideline)::

*api*
- add fun `elementsOf` to apis/fluent-en_GB/atrium-fluent_en_GB-common/.../iterableContainsInAnyOrderOnlyCreators.kt (see other functions there for guideance)
  - it is enough if you delegate to `values`
  - `require` that the `Iterable.hasNext`
  - add `@since 0.9.0` to the KDOC
- extend IterableContainsInAnyOrderOnlyValuesAssertionsSpec via inlcude (see BuilderSpec in IterableContainsInAnyOrderAtLeast1ValuesAssertionsSpec for an example) 

## Your first contribution?
- Write a comment `I'll work on this` if you would like to take this issue over. 
  This way we get the chance to revise the description in case things have changed in the meantime,
  we might give you additional hints and we can assign the task to you, so that others do not start as well.
- See [Your first code contribution](https://github.com/robstoll/atrium/blob/master/.github/CONTRIBUTING.md#your-first-code-contribution) for guidelines.  
- Do not hesitate to ask questions here or to contact us via [Atrium's slack channel](https://kotlinlang.slack.com/team/U3DE1TXKP) if you need help
  ([Invite yourself](http://slack.kotlinlang.org/) in case you do not have an account yet).

**Comment by PawelRaciborski** [comment]
I'll work on this

**Comment by robstoll** [comment]
@PawelRaciborski do you need any help?

**Comment by robstoll** [comment]
@PawelRaciborski un-assign you so that others can take over.

**Comment by lelloman** [comment]
I'll work on this

**Comment by robstoll** [comment]
@lelloman are you still working on it, do you need help?

**Comment by lelloman** [comment]
Hi @robstoll sorry I got very busy lately. I tried to implement it once but got lost, I'd like to give it another try this week and eventually ask for specific help if that's OK

**Comment by robstoll** [comment]
@lelloman any updates on this one? Still here if you need help :slightly_smiling_face:

**Comment by johnGachihi** [comment]
I'll work on this

**Comment by robstoll** [comment]
implemented with 3bee602
